.github/workflows/tests.yml

```diff
@@ -19,10 +19,9 @@ jobs:
     strategy:
       matrix:
         config:
-          - {python-version: 3.6, interfaces: ['tf']}
-          - {python-version: 3.7, interfaces: ['torch']}
+          - {python-version: 3.7, interfaces: ['tf', 'torch', 'jax']}

```

Comment:
This would add TF and Jax to 3.7 tests :thinking:

Reply:
Is it worth having the following?

```yaml
- {python-version: 3.7, interfaces: ['tf']}
- {python-version: 3.7, interfaces: ['torch']}
- {python-version: 3.8, interfaces: ['tf', 'torch', 'jax']}
- {python-version: 3.9, interfaces: []}
```

In the past, it has been useful to have tests that only run with Torch, and similar for TF